src/careamics/config/nm_model.py

```diff
@@ -16,21 +16,19 @@ class GaussianMixtureNMConfig(BaseModel):
     model_config = ConfigDict(
         validate_assignment=True, arbitrary_types_allowed=True, extra="allow"
     )
-    # TODO What are all these parameters?
     # model type
     model_type: Literal["GaussianMixtureNoiseModel"]
 
-    path: Optional[Union[Path, str]] = None
```

Comment:
Was moving the docstring before the attributes declaration done automatically or manually?

According to [PEP257](https://peps.python.org/pep-0257/#what-is-a-docstring) (and I'll grant you that it is really old..): 

> String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called “attribute docstrings”.

And that's (I believe) is the reason the class attributes/fields gave the docstring after and not before the declaration/assignment.

Now this needs to be respected in order for Pydantic to extract it correctly and make it available to users when exporting json schema or just exploring the automatically generated description of each model and attribute (see [this](https://github.com/pydantic/pydantic/pull/6563)).